### ISSUE TITLE
pottery shard cycling

### ISSUE THREAD

**Issue by Divineaspect** [codetaylor/pyrotech#203]
I would like to be able to hammer or otherwise grind (I'm using adv mortars) pottery shards into dried clay dust, which in turn can be soaked in water to regain workable clay.

**Comment by nephatrine** [comment]
Once you bake clay, you can't really turn it back into workable clay again. The high temperatures change the material pretty radically. Grinding the pottery into dust would give you basically the same thing as grinding up a rock into dust and clay formation from rock sediments is a long geologic process. So I think the sand pile you get currently from pottery shards is pretty accurate.

As far as game mechanics go, there's already a dirt clump -> clay recipe the mod adds that is simple enough if you're just looking for an easy way to obtain more clay.

**Comment by Divineaspect** [comment]
*does more research*
that's kinda interesting, because when I was a scout they had us grind it up and use it to make mortar if it broke when firing. While clay isn't the same material, I think it's the closest one in minecraft.

**Comment by codetaylor** [comment]
What problem would this solution solve? Not enough clay or not enough to do with the broken clay?

**Comment by Divineaspect** [comment]
I was thinking of both, but I'd be satisfied with resolving both of them separately.
the clay issue could be resolved by just adding a soaking pot recipe for the dirt instead of requiring iron buckets.
The only uses I can think of for pottery shards other then mortars is making mosaics or arrowheads but flint isn't exactly rare.

**Comment by nephatrine** [comment]
I don't think this belongs in pyrotech necessarily, but it'd be neat to make kintsugi repaired pottery with the shards. They are a little worthless imo otherwise.

As for making clay, yeah it would be neat to have a better way to make it from dirt, though it'd be nice if the soaking tub could return multiple outputs so soaking a dirt you could extract maybe both sand pile and clay from it since both are components of many dirts.
